.wordlist.txt

```diff
@@ -124,6 +124,7 @@ iptables
 init
 interprocess
 Javascript
+JIRA
```

Comment:
What is this for?

Reply:
Oops, I think this was a change meant for the other release notes PR and it crept it here. I will revert this in this PR as it is out of context.